scripts/categorize_scenes.py

```diff
@@ -0,0 +1,92 @@
+import argparse
```

Comment:
Add a small readme in the folder to mention how to use the script and what they do.